graphql-spring-boot-test/src/main/java/com/graphql/spring/boot/test/GraphQLTestSubscription.java

```diff
@@ -220,6 +235,19 @@ public GraphQLResponse awaitAndGetNextResponse(final int timeout) {
    * @return The received response.
    */
   public GraphQLResponse awaitAndGetNextResponse(final int timeout, final boolean stopAfter) {
+    return awaitAndGetNextResponse(Duration.ofMillis(timeout),stopAfter);
```

Comment:
I suppose all methods with the `int` variants should be deprecated.

Reply:
Thank you. It's dealt with